jhipster-jdl.jdl

```diff
@@ -306,6 +304,8 @@ entity FdaSubmissionType {
     type FdaSubmissionTypeKey required unique
     name String required
     shortName String
+    submissionPrefix String
+    submissionLink String
```

Comment:
It seems by adding the prefix and the link to FdaSubmission would be useful for the frontend to parse the info.